apps/vscode/command_client/vs_code.py

```diff
@@ -0,0 +1,22 @@
+import string
+
+from talon import Context, actions
+
+ctx = Context()
+
+ctx.matches = r"""
+app: vscode
+"""
+ctx.tags = ["user.command_client"]
+
+
+@ctx.action_class("user")
+class VsCodeAction:
+    def command_server_directory() -> string:
+        return "vscode-command-server"
+
```

Comment:
And then define the action `vscode` here, and call the `run_command` action, wrapped in a try-catch. If it throws an error, do the fallback

Reply:
Done, this but duplicated code